index.js

```diff
@@ -0,0 +1,37 @@
+import { DateTime } from './modules/luxon/src/luxon.js';
+import Navigator from './modules/navigator.js';
+import BookList from './modules/bookList.js';
+import Book from './modules/book.js';
+
+function showTime() {
+  const currentDate = DateTime.local().toFormat('yyyy-MMM-dd');
+  const currentTime = DateTime.local().toFormat('hh:mm:ss');
+  document.querySelector('.time').innerHTML = `Date: ${currentDate}  Time:${currentTime}`;
+}
```

Comment:
- [ ] Great job for the showTime()() function implementation, :clap: but I think it will be better if you can use arrow syntax for functions instead of a  function declaration for more information [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html)